### DISCUSSION TITLE
Difference between momepy.Shannon and momepy.shannon_diversity

### DISCUSSION THREAD

**Discussion by cdkang** [pysal/momepy#381] [Q&A] [answered]
Hello, 

Could you let me know difference between momepy.Shannon and momepy.shannon_diversity more clearly? 

Sincerely, 

Chang-Deok Kang

**Comment by martinfleis** [comment, marked as answer]
Hello, `shannon_diversity` is just a formula giving a scalar (single) value of Shannon diversity as an output.`Shannon` loops over your geometries and measures Shannon diversity of a given column within a local neighbourhood of each geometry as is defined in a binary spatial weights matrix. `shannon_diversity` is therefore called within `Shannon`. We just exposed it as well as it may be useful to access it directly.

**Reply by cdkang** [reply]
Thank you for your information. It is very helpful.
